pkg/storage/storage_test.go

```diff
@@ -278,8 +278,40 @@ func TestStorageHistory(t *testing.T) {
 	}
 }
 
-func TestStorageRemoveLeastRecentWithError(t *testing.T) {
-	storage := Init(driver.NewMemory())
+var ErrMaxHistoryMockDriverSomethingHappened = errors.New("something happened")
```

Comment:
This variable seems like having more visibility than it needs.
```suggestion
var errMaxHistoryMockDriverSomethingHappened = errors.New("something happened")
```

Reply:
Thanks!